### PR TITLE
[1LP][RFR] This will separate manual_tests

### PR DESCRIPTION
**Pull request by sbulage**
- Moved Manual Tests to respective automated folder.

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by mshriver** [comment]
@sbulage I'm sorry, but I have likely caused you to need a potentially significant refactor on your PR here.

In PR #8401 I merged changes to the polarion docstrings in order to meet an updated standard for required fields in the polarion-docstrings flake plugin.

Please let me know if you need any assistance completing this refactor, or would like me to sort the correct changes on your branch.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 1]
```diff
@@ -0,0 +1,1275 @@
...
+
+
+@pytest.mark.manual
+@test_requirements.ansible
+@pytest.mark.tier(2)
+def test_embed_tower_invisible():
+    """
+    Embedded Ansible Tower provider won"t be visible in the CFME UI (Tower
+    should be headless, its UI should not be enabled.) p1
+
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: @sbulage 
Is this ```p1``` typo mistake?
If not, please explain?

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 2]
```diff
@@ -0,0 +1,1275 @@
...
+    After all processes are running make sure websockets are enabled then
+    add a repo with the same name as a current repo and check the
+    notifications display correctly. With a Red banner to show it was
+    unsuccessful.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1471868
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/6h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: @sbulage 
polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 3]
```diff
@@ -0,0 +1,1275 @@
...
+@test_requirements.ansible
+def test_embed_tower_repo_tag():
+    """
+    RBAC - tag Ansible repo and allow new user see only this repo.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1526217
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/2h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 4]
```diff
@@ -0,0 +1,1275 @@
...
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.ansible
+def test_embed_tower_repo_links():
+    """
+    test clicking #of playbooks cell will navigate to the Playbooks area,
+    filtered by the associated repo name.
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: Is ```#``` typo mistake?
If yes then please remove.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 5]
```diff
@@ -0,0 +1,1275 @@
...
+    add AWS EC2 credentials
+    create new catalog
+    add new catalog item with EC2 playbook
+    order the service and wait for it"s execution
+    check if service was finished OK
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 6]
```diff
@@ -0,0 +1,1275 @@
...
+def test_embed_tower_repo_list():
+    """
+    After all processes are running add a few repo"s for playbooks. Check
+    that all these repos appear in the repo list section in the ui, With
+    the correct status and the correct quantity of playbooks.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/6h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 7]
```diff
@@ -0,0 +1,1275 @@
...
+@pytest.mark.tier(1)
+def test_embed_tower_event_catcher_collect():
+    """
+    EventCatcher process collects all activity from api/acitivity_streamis
+    and is writing data into PG DB
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/4h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 8]
```diff
@@ -0,0 +1,1275 @@
...
+    Allow user/admin to create/import credentials for machines which will
+    be managed (may need to be split into multiple tests to cover
+    -Machine, Network, Amazon Web Services, Rackspace, VMware vCenter, Red
+    Hat Satellite 6, Red Hat CloudForms, Google Compute Engine, Microsoft
+    Azure Classic, Microsoft Azure Resource Manager, OpenStack)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 9]
```diff
@@ -0,0 +1,1275 @@
...
+@test_requirements.ansible
+@pytest.mark.tier(1)
+def test_embed_tower_add_gce_credentials():
+    """
+    Add GCE credentials.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/2h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 10]
```diff
@@ -0,0 +1,1275 @@
...
+@test_requirements.ansible
+@pytest.mark.tier(1)
+def test_embed_tower_add_amazon_credentials():
+    """
+    Add Amazon credentials.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/2h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 11]
```diff
@@ -0,0 +1,1275 @@
...
+    """
+    After all processes are running and websockets role is enabled, add a
+    new repo to embedded tower and check the notifications display
+    correctly. With a Green banner to show it was successful.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1471868
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/6h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 12]
```diff
@@ -0,0 +1,1275 @@
...
+def test_embed_tower_creds_details():
+    """
+    Clicking on a cred name should show details of the Credentials.
+    (Automation-Ansible-Credentials Table view showing provider creds
+    added)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/6h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 13]
```diff
@@ -0,0 +1,1275 @@
...
+def test_embedded_ansible_update_bad_version_59017():
+    """
+    Tests updating an appliance which has embedded ansible role enabled,
+    also confirms that the
+    role continues to function correctly after the update has completed
+    Test Source
+
+    Polarion:
+        assignee: None
+        initialEstimate: None
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tags ```caseimportance:```, ```casecomponent:``` are missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 14]
```diff
@@ -0,0 +1,1275 @@
...
+@pytest.mark.tier(1)
+def test_embed_tower_add_machine_credentials_vault():
+    """
+    Add vault password and test in the playbook that encrypted yml can be
+    decrypted.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1/2h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 15]
```diff
@@ -0,0 +1,1275 @@
...
+    Allow user/admin to create/import credentials for machines which will
+    be managed (may need to be split into multiple tests to cover
+    -Machine, Network, Amazon Web Services, Rackspace, VMware vCenter, Red
+    Hat Satellite 6, Red Hat CloudForms, Google Compute Engine, Microsoft
+    Azure Classic, Microsoft Azure Resource Manager, OpenStack)
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: ansible
+        initialEstimate: 1h
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 16]
```diff
@@ -0,0 +1,1275 @@
...
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 17]
```diff
@@ -0,0 +1,1275 @@
...
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: polarion tag ```caseimportance:``` is missing.

**Comment by sbulage** [comment]
Travis is failing b/c there are lot many tests present without, Assignee, initialEestimate, Component names.

**Comment by mshriver** [comment]
Looks like you should re-try the rebase, all of those missing fields in test_master were fixed in #8401

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 18]
```diff
@@ -0,0 +1,1211 @@
+import pytest
+
+from cfme import test_requirements
+
+
+@pytest.mark.manual
+@test_requirements.ansible
+def test_embed_tower_exec_play_against_vmware():
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: @sbulage 
Please add ```tier``` for this test case.
Also take a look at other tests and update with ```@pytest.mark.tier()```

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_manual.py]: @ganeshhubale Added.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 19]
```diff
@@ -0,0 +1,1211 @@
...
+    18. Wait until the service will be provisioned.
+    19. Navigate tot Services/My Services.
+    20. Open details of the provisioned service, open "Provisioning" tab.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        initialEstimate: 1/2h
+        tags: ansible_embed
+    """
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: Please add above ```testSteps``` under polarion tags.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 20]
```diff
@@ -0,0 +1,2650 @@
...
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_groups_azure_ubuntu():
+    """
+    1. Add Azure provider
+    2. Perform SSA on Ubuntu instance.
+    3. Check Groups are retrieved.
+
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: add ```testSteps``` under polarion tags.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 21]
```diff
@@ -0,0 +1,2650 @@
...
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_groups_azure_rhel():
+    """
+    1. Add Azure provider
+    2. Perform SSA on RHEL instance.
+    3. Check Groups  are retrieved.
+
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: add ```testSteps``` under polarion tags.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 22]
```diff
@@ -0,0 +1,2650 @@
...
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(1)
+def test_ssa_groups_azure_windows2012r2_ntfs():
+    """
+    1. Add Azure provider
+    2. Perform SSA on Windows server 2012 R2.
+    3. Check Groups are retrieved.
+
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: add ```testSteps``` under polarion tags.
Also update other other test cases which are having same issue.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 23]
```diff
@@ -0,0 +1,2650 @@
...
+@pytest.mark.tier(1)
+def test_ssa_vm_azure_managed_disk():
+    """
+    Perform SSA on Managed disk on Azure provider.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        initialEstimate: 1/2h
+    """
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: Please add ```tags```

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 24]
```diff
@@ -0,0 +1,2650 @@
...
+def test_ssa_vm_users():
+    """
+    Check users are fetched correctly for analysed VM
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        initialEstimate: 1/2h
+        testtype: integration
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: ```tags:``` missing.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 25]
```diff
@@ -0,0 +1,2650 @@
...
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: SmartState
+        caseimportance: medium
+        caseposneg: negative
+        initialEstimate: 1/4h
+        setup: Need to create a windows 2016 instance and save it onto the ReFS disks
+               on Azure.
+               Add a second disk to the VM config.  Open VM, initialize disk and
```
Comment (ganeshhubale) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: ```setup:``` should be like
```
 setup:
     1. Need to create a windows 2016 instance and save it onto the ReFS disks
         on Azure.
     2.
```

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 24 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: Tags added.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 21 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: I will update it while Automating tests.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 26]
```diff
@@ -0,0 +1,1213 @@
...
+    Playbook Service Dialog is built, it has default parameters, which can
+    be overridden at "ordering" time. Check if the overridden parameters
+    are passed.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
+        caseimportance: medium
+        initialEstimate: 1/6h
+    """
```
Comment (ganeshhubale) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: add ```tags:```

Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_manual.py]: added.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 25 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: This is present for almost all tests, I will re-format it while automating them.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 27]
```diff
@@ -0,0 +1,1214 @@
...
+        assignee: sbulage
+        casecomponent: Ansible
+        initialEstimate: 1/4h
+        tags: ansible_embed
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.ansible
```
Comment (valaparthvi) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: Tier Missing

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 28]
```diff
@@ -0,0 +1,2773 @@
...
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+@pytest.mark.manual
+@test_requirements.smartstate
+@pytest.mark.tier(2)
```
Comment (valaparthvi) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: You are using tier(2) only in this case, every other test case uses tier(1), so you can add tier mark to global pytestmark.

File: cfme/tests/ansible/test_embedded_ansible_manual.py (hunk 27 above)
Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_manual.py]: Added it.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 28 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: I have added global tier for it.

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 29]
```diff
@@ -0,0 +1,1090 @@
...
+
+
+@pytest.mark.tier(1)
+def test_embed_tower_add_scm_credentials():
+    """
+    Add SCM credentials for private GIT repo.
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
```
Comment (jawatts) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: missing `caseimportance` here

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 30]
```diff
@@ -0,0 +1,1090 @@
...
+
+
+@pytest.mark.tier(2)
+def test_embed_ansible_next_gen():
+    """
+    Follow BZ: https://bugzilla.redhat.com/show_bug.cgi?id=1511126
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
```
Comment (jawatts) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: missing `caseimportance`

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 31]
```diff
@@ -0,0 +1,1090 @@
...
+    """
+    After all processes are running make sure websockets are enabled then
+    add a repo with the same name as a current repo and check the
+    notifications display correctly. With a Red banner to show it was
+    unsuccessful.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1471868
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
```
Comment (jawatts) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: `caseimportance`

File: cfme/tests/ansible/test_embedded_ansible_manual.py [hunk 32]
```diff
@@ -0,0 +1,1090 @@
...
+
+@pytest.mark.tier(3)
+def test_embed_tower_repo_tag():
+    """
+    RBAC - tag Ansible repo and allow new user see only this repo.
+    https://bugzilla.redhat.com/show_bug.cgi?id=1526217
+
+    Polarion:
+        assignee: sbulage
+        casecomponent: Ansible
```
Comment (jawatts) [inline cfme/tests/ansible/test_embedded_ansible_manual.py]: `caseimportance`

File: cfme/tests/ansible/test_embedded_ansible_manual.py (hunk 29 above)
Comment (sbulage) [reply cfme/tests/ansible/test_embedded_ansible_manual.py]: If nothing specified it will assume to be High, right .?

Comment (jawatts) [reply cfme/tests/ansible/test_embedded_ansible_manual.py]: @sbulage you are correct, ignore me

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 33]
```diff
@@ -0,0 +1,2350 @@
...
+        initialEstimate: 1/3h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_groups_rhos7_ga_fedora_22_ext4():
+    """
+    test_ssa_groups[rhos7-ga-fedora-22-ext4]
```
Comment (jawatts) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: was this suppose to be updated?

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 34]
```diff
@@ -0,0 +1,2350 @@
...
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_vm_vsphere6_nested_centos_xfs_ssui():
+    """
+    1. Provision service with CentOS VM
+    2. Perform SSA on it.
+    3. Check data populated on Provisioned Service in SSUI Dashboard.
```
Comment (jawatts) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: this meant to be numbered?

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 35]
```diff
@@ -0,0 +1,2350 @@
...
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_vm_ec2_windows2016_ssui():
+    """
+    1. Provision service with Windows 2016 VM
+    2. Perform SSA on it.
+    3. Check data populated on Provisioned Service in SSUI Dashboard.
```
Comment (jawatts) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: same comment regarding numbering

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py [hunk 36]
```diff
@@ -0,0 +1,2350 @@
...
+        initialEstimate: 1/2h
+        startsin: 5.6
+        tags: smartstate
+    """
+    pass
+
+
+def test_ssa_packages_rhos7_ga_fedora_22_ext4():
+    """
+    test_ssa_packages[rhos7-ga-fedora-22-ext4]
```
Comment (jawatts) [inline cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: Was this suppose to be a description here?

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 33 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: I am removing those tests.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 34 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: Kept intentionally, will remove while automating those.

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by sbulage** [comment]
I don't want more Rebase for this PR :(

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 35 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: Refer above comments.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 36 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: Removed tests already automated.

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py (hunk 33 above)
Comment (sbulage) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis_manual.py]: Removed.

**Comment by sbulage** [comment]
@mshriver can please you merge this .?

**Pull request merged**